packages/telemetry/src/slog-to-otel.js

```diff
@@ -501,60 +623,155 @@ export const makeSlogToOtelKit = (tracer, overrideAttrs = {}) => {
             },
             options,
           );
+        const syscallType = ksc[0];
+        const name = `syscall-${syscallType}`;
         // TODO: get type from packages/SwingSet/src/kernel/kernelSyscall.js
-        switch (ksc[0]) {
+        const detailPrefix = `s-${attrs.vatID}`;
```

Comment:
Wondering if the prefix should be more unique. E.g. `s-${crankNum}` (if we have the crankNum)

Reply:
Yeah, that's how you laid it out in your notes.  However, my experience was that the crankNum wasn't always available.  It could be made available with some additional effort, but I was trying to minimize the volume of work.